packages/tsconfig/src/read-tsconfig.ts

```
@@ -208,12 +251,15 @@ const internalParseTsConfig = (tsconfigPath: string, options?: Options, circular
         config.compilerOptions = {};
     }
 
-    if (config.files) {
-        config.files = config.files.map((element) => normalizePath(element));
-    }
-
     if (config.include) {
         config.include = config.include.map((element) => normalize(element));
+
+        if (config.files) {
+            delete config.files;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)